README.md

```diff
@@ -1,24 +1,53 @@
 # Global, World Scale Multiplayer Game Demo
 
+![Screenshot of the game](images/game.png)
+
+## Overview
+
 This multiplayer demo is a cloud first implementation of a global scale, realtime multiplayer game utilising
 dedicated game servers, utlising both Google Cloud's products and open source gaming solutions.
 
-## OAuth Authentication
+Projects and products utilised include:
 
-We need to manually set up the OAuth authentication, as unfortunately this cannot be automated.
+*   [Unreal Engine 5](https://www.unrealengine.com/) for the game client and server code.
```

Comment:
@kennycoder I lifted this from the draft I wrote in the blog post - WDYT?

Reply:
SGTM!